src/MCPClient/tests/test_dip_generation_helper.py

```diff
@@ -14,7 +14,7 @@
 
 class TestParseArchivesSpaceIDs(TestCase):
 
-    fixture_files = ['sip.json', 'files.json', 'archivesspaceconfig.json', 'archivesspace.json']
+    fixture_files = ['sip.json', 'files.json', 'archivesspaceconfig.json']
```

Comment:
This fixture was removed because it just contained database workflow data that is being removed.